pubspec.yaml

```diff
@@ -4,13 +4,12 @@ version: 0.0.1
 author: Mitch Hymel <mitchell.hymel@gmail.com>
 homepage: https://github.com/mitchhymel/local_notifications
 
+
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.1.1
+  meta: any
 
-environment:
-    sdk: ">=1.8.0 <2.0.0"
```

Comment:
This is required in order to publish the library to pub: https://www.dartlang.org/tools/pub/publishing
You can test with:
```
pub publish --dry-run
```
Though I'm not sure what will happen since I'm the package owner.